.gitignore

```diff
@@ -45,4 +45,6 @@ Thumbs.db
 src/api.config.ts
 version.js
 yarn-error.log
-.vscode
\ No newline at end of file
+.vscode
+/temp/*
```

Comment:
yes,  reverted the file, I guess i shouldn't check it in then only keep it checked out in my folder?